### PR TITLE
Alignment using bowtie 

### PR DESCRIPTION
**Pull request by CalounovaT**
* Runs the trinity script for aligning the reads using bowtie2. Uses the strand specific information also for rnaspades.
`snakemake align_reads`

* Then runs sorting and indexing with samtools for possible visualization in IGV.
`snakemake IGV`

* Fixes missing comma in all rule

* Fixes typo: conda instal instead of conda install

* updates readme for necessary rsem package

* updates readme for how to run the alignment and how to visualize the results in IGV.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +137,21 @@ cd tests
...
+snakemake align_reads --cores 16 
+snakemake IGV --cores 4
+~~~~
+
+Then you can load your transcriptome file to IGV: Genomes -> Load Genome from File -> select the file *transcriptome.fasta*
+
+Your sorted .bam files will be in the bowtie_alignments folder: 
+~~~~
+bowtie_alignments/{sample}.sorted.bam
+bowtie_alignments/{sample}sorted.bam.bai
```
Comment (tomas-pluskal) [inline README.md:152]: `{sample}sorted.bam.bai` or `{sample}.sorted.bam.bai`?

Comment (CalounovaT) [reply README.md:152]: Both types of files (.bam and .bam.bai) will be in the folder, in IGV one only uploads the .bam file but .bam.bai (with same file name) needs to be present in the same folder.

Comment (tomas-pluskal) [reply README.md:152]: I was just pointing to the missing period in `{sample}sorted.bam.bai`

Comment (CalounovaT) [reply README.md]: Ohh I did not read properly... It is fixed now.

Comment (tomas-pluskal) [reply README.md]: 👍🏻

**Comment by tomas-pluskal** [comment]
Thanks!

**Pull request merged**